Fix pingpong teal test

egieseke opened the pull request:
## Summary

Updated the netgoal utility to only create one wallet per non-participating node to better concentrate Algos for use by pingpong. 

## Test Plan

Ran netgoal to update the scenario2 recipe and tested the result with the CICD performance pipeline.

tsachiherman reviewed (dismissed):
looks good to me. Please make sure to regenerate all the scenarios with this change and include them in the PR.

tsachiherman commented:
Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?

onetechnical commented:
> Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?

+1. I'm still not a fan of increasing the amount of supply, as it indicates an unrealistic test (we would never increase mainnet supply).

egieseke commented on test/testdata/deployednettemplates/recipes/scenario2/genesis.json:
```diff
@@ -8,1077 +8,1077 @@
 		{
 			"Name": "Wallet207",
-			"Stake": 0.46511627906976744,
+			"Stake": 3.3333333333333335,
```
The rebalanced stake is computed in the function 
`func generateWalletGenesisData(wallets, npnHosts int)`
in `cmd/netgoal/generate.go`

algobarb reviewed (dismissed):
Messaged Eric to learn more about the details. The results look correct on Datadog now so I think the last fix to rebalance stake fixed it! Changes look good to me.

egieseke commented:
> looks good to me. Please make sure to regenerate all the scenarios with this change and include them in the PR.

Updated scenario1 and scenario3.

egieseke commented:
> Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?

By rebalancing the stake between participating and non-participating nodes, increasing the total stake is not required.  Reverted total stake back to 10 billion Algos.

algobarb reviewed (approved):
Looks like Scenario1 results look ok with the rebalanced stake and removed wallets as well. Looks good to me.

egieseke commented:
> > Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?
> 
> +1. I'm still not a fan of increasing the amount of supply, as it indicates an unrealistic test (we would never increase mainnet supply).

@onetechnical Good point.  Updated so that the total stake remains at 10 billion.  Split stake evenly between participating and nonparticipating nodes, which provides a sufficient amount for the teal pingpong tests.

bricerisingalgorand commented on cmd/netgoal/generate.go:
```diff
@@ -465,19 +466,29 @@ func saveGoalTemplateToDisk(template netdeploy.NetworkTemplate, filename string)
+	nonParticipatingNodeStake := ratZero
 	if npnHosts > 0 {
-		wallets = 2 * wallets
+		// split participating an non participating stake evenly
```
Have we always been assuming an even split between participating and non participating nodes? I wonder if that should be configurable. It looks like that would be a new feature though so probably out of scope
